com.obsproject.Studio.Plugin.Backgroundremoval.metainfo.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="addon">
+  <id>com.obsproject.Studio.Plugin.Backgroundremoval</id>
```

Comment:
Would you mind capitalizing this as `com.obsproject.Studio.Plugin.BackgroundRemoval` instead? Please update the paths in the file to reflect that too

Reply:
> Would you mind capitalizing this as `com.obsproject.Studio.Plugin.BackgroundRemoval` instead? Please update the paths in the file to reflect that too

@GeorgesStavracas shuold be all set now